config/locales/en.yml

```diff
@@ -12,40 +12,30 @@ en:
   home: Home
 
   add: Add
-  update: Update
   view: View
   edit: Edit
+  update: Update
   delete: Delete
 
   yes: Yes
   no: No
   are_you_sure: Are you sure?
 
-  settings:
-    admin_ip_list: Admin IP list
-    default_page: Default page
-    default_section: Default section
-
-    features:
-      users:
-        can_register: Allow users to register
-        can_login: Allow users to login
-
 
   # ========== ( Main Site ) ==========
 
+  features:
```

Comment:
Rails i18n supports variable interpolation which allows for:

```yaml
features:
  alert: "Sorry, the %{feature_name} feature of this site has been turned off"
```

```rb
I18n.translate('features.alert', feature_name: 'Foo')
#> "Sorry, the Foo feature of this site has been turned off"
```

Search for mentions of "interpolat" in https://guides.rubyonrails.org/i18n.html for more details.